test/tests/InterpreterTest.cpp

```diff
@@ -199,6 +199,31 @@ TEST_F(InterpreterTest, check_that_response_template_is_adapted)
 }
 
 
+TEST_F(InterpreterTest, check_that_multiple_lines_are_parsed)
```

Comment:
It would be great to have a test case where an intent has several entities over multiple lines to check that the multi line is correctly treated as one intent.